cms/static/sass/_build.scss

```diff
@@ -67,6 +67,7 @@
 @import 'views/group-configuration';
 @import 'views/video-upload';
 @import 'views/certificates';
+@import 'views/creative-commons';
```

Comment:
Is the CC stuff really specifically tied to a particular view? If so, placing it under the ``views/`` dir is cool, If not and used throughout several pages, it should be a component.